components/organisms/o-search-panel.vue

```diff
@@ -228,9 +228,7 @@ export default {
       flex-wrap: wrap;
     }
     &__product-card {
-      --product-card-max-width: 200px;
-      flex: 0 1 25%;
-      min-width: calc(var(--product-card-max-width) * 0.8);
+      flex: 0 1 50%;
```

Comment:
Did you test this for desktop? Looks like this will force the desktop view to only show 2 items in a row. The actual fix for the iPhone X screen view is to remove the mobile specific padding at line 187.